_posts/2022-01-07-The-Basics-of-Machine-Learning-without-math/2022-01-07-The-Basics-of-Machine-Learning-without-math.md

```diff
@@ -0,0 +1,66 @@
+---
+layout: post
+title:  "The Basics of Machine Learning without math"
+date:   2022-01-07 19:58:49 +0100
+categories: machine learning
+---
+
+
+## Table of Contents
+1. How is a neuronal network structured
+2. What is a neuron
+3. What is Backpropagation
+4. How can we use this knowledge in practice
+
+
+## How is a neuronal network structured
+For many people a neuronal network is magic. If you never learnt what happens behind the scenes it looks like you just throw things in and eventually it can solve a problem <br>
+A neuronal Network, as the name already suggests, is a network of mutliple neurons.
+
+
+<img src="1200px-Neural_network.svg.png" width="600"><br>
+https://upload.wikimedia.org/wikipedia/commons/thumb/3/3d/Neural_network.svg/1200px-Neural_network.svg.png <br><br>
+
+
+
+
+You give a input into it and through the interaction of the different neurons the result is formed
+
+## What is a Neuron
+
+As you could see in the image above there are 3 types of neurons.
+The green ones are input Neurons. The just have connections to their right and no other connections. These are used to input data into the neuronal network. <br>
+The second type are the blue ones. How many collumns of them exist is not given and can be changed. They have connections to their left (input) and to their right (output)
+They take the inputs, process them and then generate an output. <br>
+The last type of neurons are the yellow one. They are at the end of the network. Their Output is the output of the whole network. <br>
+
+
+All 3 types of the network can be descriped through one model
+
+<img src="1*O7YSSqlOdQuNgMrH7J_3dg.png" width="600"><br>
+https://miro.medium.com/max/428/1*O7YSSqlOdQuNgMrH7J_3dg.png<br><br>
+
+A neuron has multiple inputs. The inputs are the outputs of all neurons in the previous collum.
+These inputs get weights. For example if a input is 0.6 and the weight is 0.1 we take 0.6 * 0.1 and then we have the "real" input. Every input has its own weight. All of the inputs multiplied with the weigth are added together. This sum is now put into a function. The function determines how much output is generated for the given sum. Often the function looks like this:
+
+<img src="1*oePAhrm74RNnNEolprmTaQ.png" width="600"><br>
+https://miro.medium.com/max/357/1*oePAhrm74RNnNEolprmTaQ.png<br><br>
```

Comment:
Same problem here with the license. Source seems to be https://medium.com/analytics-vidhya/neural-network-part1-inside-a-single-neuron-fee5e44f1e again